src/config/CemuConfig.cpp

```diff
@@ -89,12 +89,12 @@ void CemuConfig::Load(XMLConfigParser& parser)
 	auto gamelist = parser.get("GameList");
 	game_list_style = gamelist.get("style", 0);
 	game_list_column_order = gamelist.get("order", "");
-	column_width.name = gamelist.get("name_width", -3);
-	column_width.version = gamelist.get("version_width", -3);
-	column_width.dlc = gamelist.get("dlc_width", -3);
-	column_width.game_time = gamelist.get("game_time_width", -3);
-	column_width.game_started = gamelist.get("game_started_width", -3);
-	column_width.region = gamelist.get("region_width", -3);
+	column_width.name = std::max(gamelist.get("name_width", column_width.name), 0u);
```

Comment:
Please don't merge for now. This actually needs fixing.
tinyxml (and XMLConfig.h) seem to silently cast the `-3` to an unsigned...